packages/taquito-local-forging/src/proto12-ithaca/schema.ts

```diff
@@ -36,6 +37,6 @@ export const operationDecoderProto12 =
         ...decodedObj,
       };
     } else {
-      throw new Error(`Unsupported operation ${op[0]}`);
+      throw new UnsupportedOperationError(op[0].toString());
```

Comment:
When there is data manipulation like `toString` in this case, I think it would be valuable to add a unit test just to make sure that the message renders as expected. (This applies to other errors following a similar pattern)